### PR TITLE
(MAINT) Change how rsync copies rpm & deb repos

### PR DESCRIPTION
**Pull request by mckern**
These new settings should hopefully lead to less conflict
and gnashing of teeth and rending of garments and lamentations
from our loved ones.

### CODE REVIEW

**Comment by ScottGarman** [comment]
Short of trying to ship something using this change, how else can it be tested? I'm cautiously :+1:

**Comment by mckern** [comment]
I have some new test/dev servers that we can test this against. I'll get some data tonight.

File: lib/packaging/deb/repo.rb [hunk 1]
```diff
@@ -194,16 +194,21 @@ def repo_deployment_command(origin_path, destination_path, destination, dryrun =
       # fail due to permission errors.
       options = %w(
         rsync
+        --ignore-existing
+        --recursive
+        --links
         --hard-links
-        --copy-links
```
Comment (underscorgan) [inline lib/packaging/deb/repo.rb]: I think we still need --copy-links

**Comment by haus** [comment]
I love that the flags needed are so different.

**Comment by mckern** [comment]
tumblr_n3cochlvij1sl4pv2o1_500

File: lib/packaging/deb/repo.rb (hunk 1 above)
Comment (mckern) [inline lib/packaging/deb/repo.rb]: Oof, we do; we're still resolving those debs aren't we? OK. Will fix.

**Comment by stahnma** [comment]
@mckern  so this is 24 days old. Is it ready or abandoned?

**Comment by mckern** [comment]
@stahnma neither but I don't think we ever got agreement on what the filesystem on the receiving end should absolutely look like, so this didn't go anywhere. Closing for now, will reopen if we nail that down.

**Pull request closed without merging**